messages.json

```diff
@@ -53,7 +53,8 @@
     "roles_messages": [
         "_ _\nReact to this message to get pronoun roles\n:regional_indicator_h: - He/Him\n:regional_indicator_s: - She/Her\n:regional_indicator_t: - They/Them",
         "_ _\nReact to this message to get year group roles\n:zero: - Foundation Year\n:one: - First Year\n:two: - Second Year\n:regional_indicator_f: - Final Year (incl. 3rd Year MSci/MEng)\n:regional_indicator_i: - Year in Industry\n:regional_indicator_a: - Year Abroad\n:regional_indicator_t: - Post-Graduate Taught (Masters/MSc)\n:regional_indicator_r: - Post-Graduate Research (PhD)\n:a: - Alumnus\n:regional_indicator_d: - Postdoc",
-        "_ _\nReact to this message to join the **opt in channels**\n:speech_balloon: - Serious Talk\n:house_with_garden: - Housing\n:video_game: - Gaming\n:tv: - Anime\n:soccer: - Sport\n:briefcase: - Industry\n:pick: - Minecraft\n:globe_with_meridians: - <Group_Name> Website\n:bookmark: - Archivist",
+        "_ _\nReact to this message to join the **opt in channels**\n:speech_balloon: - Serious Talk\n:house_with_garden: - Housing\n:video_game: - Gaming\n:tv: - Anime\n:soccer: - Sport\n:briefcase: - Industry\n:pick: - Minecraft\n:globe_with_meridians: - <Group_Name> Website\n:bookmark: - Archivist", 
+        "_ _\nReact to this message to **__opt out__** of the #starboard\n:no_entry_sign: - Messages you send **won't** be eligible to be added to the starboard, and stars you add to a message won't be counted towards the starboard threshold\n:star: - Your messages **can** be added to the starboard and stars you add to messages **will** be counted towards the starboard threshold",
```

Comment:
I suggest that the positive side doesnt include the `and stars you add to messages **will** be counted towards the starboard threshold"` as it is a bit verbose. It doesnt need to be said if the negative has already been mentioned. I think it will be clearer to not repeat that.

Reply:
This might just be me but I do think it's worth having it as clear as possible, yeah you can infer from one what the other means but I think generally being as explicit as possible is better. Happy to be overridden though if people think otherwise

Reply:
I think it's perfectly fine as is - I wouldn't want to change it to have less detail.